src/group/components/GroupEdit.vue

```diff
@@ -286,4 +372,13 @@ export default {
 
 <style scoped lang="stylus">
 @import '~editbox'
+
+.q-field--outlined >>> .q-field__control:before {
+  border 1px solid rgba(0, 0, 0, 0.15)
+}
+
+.q-field--outlined.q-field--highlighted >>> .q-field__control:not(.text-negative):after {
+  border-color rgba(0, 0, 0, 0.45)
+}
```

Comment:
Is there no way to specify this via quasar props? The quasar special classes change over time, so one day this might break and we might not easily notice.

... it does look like the type of declaration that would not be available via quasar props though. I wonder if there is a compromise approach that is _nearly_ as nice, but more standard?

Reply:
We really searched a lot but didn't found a prop or a stylus variable to tackle this. Just goggled again and saw someone suggested to use the `color` prop of the QField. We'll try that. Maybe this could be a compromise.
Else, we see no other way doing it, sadly.
Maybe a dedicated unit test that checks the QField nodes would help in this case.

Reply:
We removed the styling in favor of the default quasar styling. This has slightly darker colors but good too.